dev/com.ibm.ws.jca_fat_regr/fat/src/com/ibm/ws/jca/fat/regr/util/HttpHelper.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.jca.fat.regr.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Vector;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import componenttest.topology.impl.LibertyServer;
+
+public class HttpHelper {
```

Comment:
I don't think this class is used anywhere in your test classes.  
And if it is it can be replaced with functionality from `FATServletClient`

Reply:
removed